client/app/queue/utils.js

```diff
@@ -347,6 +347,7 @@ export const prepareAppealForStore = (appeals) => {
       nodDate: appeal.attributes.nod_date,
       certificationDate: appeal.attributes.certification_date,
       powerOfAttorney: appeal.attributes.power_of_attorney,
+      cavc_remand: appeal.attributes.cavc_remand,
```

Comment:
allow access to the remand via Redux state

Reply:
Small gripe, let's camel case this bad boy
```suggestion
      cavcRemand: appeal.attributes.cavc_remand,
```
Will need updating elsewhere on the front end I assume